src/AppHead.tsx

```diff
@@ -19,10 +20,10 @@ interface AppHeadProps {
   importElk: boolean;
   description: string;
   importPrettier: boolean;
-  ogImageUrl: string | undefined;
+  ogImageUrl: string | null;
```

Comment:
changed to null because now this is sent in the SSRed props and undefined is not JSON-serializable